### PR TITLE
Make it easier to help out

### PR DESCRIPTION
**Pull request by Hex-4**
I've added some info for people looking to contribute or just catch up. This includes a roadmap and TLDR.

## Summary by CodeRabbit

- **New Features**
  - Introduced a `ROADMAP.md` file outlining project milestones and ongoing tasks for better contributor guidance.
  - Added a `TLDR.md` file detailing the app's vision, security features, and user authentication processes.

- **Documentation**
  - Updated `CONTRIBUTING.md` to enhance clarity, including minor formatting changes and a new direction to the `ROADMAP.md`.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 <!-- omit in toc -->
```
Comment (jaspermayone) [inline CONTRIBUTING.md]: ?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,10 +1,12 @@
...
-First off, thanks for taking the time to contribute! ❤️
+First off, thanks for taking the time to contribute! 💜
 
 All types of contributions are encouraged and valued. See the [Table of Contents](#table-of-contents) for different ways to help and details about how this project handles them. Please make sure to read the relevant section before making your contribution. It will make it a lot easier for us maintainers and smooth out the experience for all involved. The community looks forward to your contributions. 🎉
 
+For a list of things that we're currently working on (and that you can help with!) check out `ROADMAP.md`.
+
 > And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
 > - Star the project
 > - Post about it on social media
```
Comment (jaspermayone) [inline CONTRIBUTING.md]: Not really an avenue we would think about... not sure how I feel here.

File: ROADMAP.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+This is a list of what the Purple Bubble project has done, and what still needs to be done.
+
+> **Contributors:** Feel free to help out with any of these tasks! The Purple Bubble team & community communicates via Slack. You can join the slack at https://join.slack.com/t/thepurplebubble/signup. If you can't join or would prefer not to, reach out to team@purplebubble.org or open a github issue if you have questions or need help.
```
Comment (jaspermayone) [inline ROADMAP.md]: Let's add PR templates for helping with common tasks

File: TLDR.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+
+
+# Introduction
+
+Hey! This is Purple Bubble. We're a crew of teens on a mission to make messaging epic again. Zero tracking, spying, or data selling. Everything is end-to-end encrypted, signed, and decentralized. All open source.
```
Comment (jaspermayone) [inline TLDR.md]: Is this the same copy from the website?

File: TLDR.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+> We mostly communicate through the [Purple Bubble Slack](https://join.slack.com/t/thepurplebubble/signup), but you can also email us (team@purplebubble.org) or open a Github issue.
+ 
+
+## Protocol
+
+Protocol v1.0.0 Jasper has multiple REST API servers in a mesh network. Clients pick two servers. One is used for transmission (TX) and the other is used for reception (RX). For faster messaging users can switch to Transparent Mode, which uses an encrypted STUN server to establish a WebRTC connection. Messages are signed by users' keys.
+
+## App
+
+We currently plan to have a desktop and mobile app. You'll be able to sign in using a key file that holds your public and private key, and your friends can add you by importing a contact file that holds your public key, name, and preferred profile picture.
```
Comment (jaspermayone) [inline TLDR.md:21]: Let's not get into feature specifics here.

**Review by jaspermayone — CHANGES_REQUESTED** [review]
Left some requests for changes.

Overall the copy will need tidying post merge, I addressed the glaring ones here

File: TLDR.md (hunk 4 above)
Comment (Hex-4) [reply TLDR.md]: yes, but from memory

Comment (jaspermayone) [reply TLDR.md]: Can we make sure it matches what's on the website for consistency?

Comment (jaspermayone) [reply TLDR.md]:
```suggestion
# CALLING ALL BUBBLERS!
We're Purple Bubble, a crew of teens on a mission to make messaging epic again! Tired of feeling like the product in those other apps? We get it. That's why we're building something new: an app that puts YOU in control. No ads, no data mining, no creepy algorithms. Just you, your friends, and a whole lot of fun.

# READY TO PEEK BEHIND THE CURTAIN?
We're in the super-early stages (think building a pillow fort!), and we're looking for awesome testers to join our inner circle. Sign up below to get the scoop on all things Purple Bubble (promise it'll be way more exciting than your dentist's newsletter - no spam, just 4-5 super cool updates a year).


```

Comment (jaspermayone) [reply TLDR.md]: this text, but change the newsletter bit to direct people to the website.

Comment (Hex-4) [reply TLDR.md]: should I direct people to sign up via the website or just to the website?

File: CONTRIBUTING.md (hunk 1 above)
Comment (Hex-4) [reply CONTRIBUTING.md]: toc was probably autogenerated so that's there, should I remove them?

Comment (jaspermayone) [reply CONTRIBUTING.md]: yes pls!

File: TLDR.md (hunk 4 above)
Comment (jaspermayone) [reply TLDR.md]: Direct to sign up via website, traffic there is good too! Our slack should be linked in these docs as well tho

File: CONTRIBUTING.md (hunk 2 above)
Comment (Hex-4) [reply CONTRIBUTING.md]: are you referring to the thing about social media?
if so i don't really like it either

Comment (jaspermayone) [reply CONTRIBUTING.md]: yes

Comment (jaspermayone) [reply CONTRIBUTING.md]: lets remove it.

File: TLDR.md (hunk 5 above)
Comment (Hex-4) [reply TLDR.md:21]: please elaborate

Comment (Luke-Oldenburg) [reply TLDR.md:21]: imo it's fine

File: ROADMAP.md (hunk 3 above)
Comment (Hex-4) [reply ROADMAP.md]: shouldn't those go into a template repo? I can make a PR there

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -50,12 +51,12 @@ We will then take care of the issue as soon as possible.
 
 ## I Want To Contribute
 
-> ### Legal Notice <!-- omit in toc -->
+> ### Legal Notice 
 > When contributing to this project, you must agree that you have authored 100% of the content, that you have the necessary rights to the content and that the content you contribute may be provided under the project license.
```
Comment (jaspermayone) [inline CONTRIBUTING.md:55]: We need to look into legal things before adding them, I will look at this later.

File: ROADMAP.md (hunk 3 above)
Comment (jaspermayone) [inline ROADMAP.md]: People need to email me to get an invite or have someone already there add them

**Review by jaspermayone — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 6 above)
Comment (Hex-4) [reply CONTRIBUTING.md:55]: remove for now?

File: ROADMAP.md (hunk 3 above)
Comment (Hex-4) [reply ROADMAP.md]: at what email address?

File: CONTRIBUTING.md (hunk 6 above)
Comment (jaspermayone) [reply CONTRIBUTING.md:55]: Yes, we will add legal stuff to all the repos at some point in the future when we need it.

File: ROADMAP.md (hunk 3 above)
Comment (jaspermayone) [reply ROADMAP.md]: jasper@purplebubble.org, or if they know someone already in the slack anyone can invite them.

Comment (Luke-Oldenburg) [reply ROADMAP.md]: should we have a contact group setup on workspaces?

File: CONTRIBUTING.md (hunk 6 above)
Comment (Luke-Oldenburg) [reply CONTRIBUTING.md:55]: Licenses cover this sort of stuff. People should read those if they want to get into the legality of what they're doing. It is a nice disclaimer to say not to add stolen code though.

File: ROADMAP.md (hunk 3 above)
Comment (jaspermayone) [reply ROADMAP.md]: we have team@purplebubble.org, we could use that instead!

Comment (jaspermayone) [reply ROADMAP.md]: I could also set up an email forward to pipe some email into slack

Comment (jaspermayone) [reply ROADMAP.md]: slack@purplebubble?

File: CONTRIBUTING.md (hunk 6 above)
Comment (jaspermayone) [reply CONTRIBUTING.md:55]: Fair enough, I just don't want to add random legal notices if we don't know what we're doing

**Comment by Luke-Oldenburg** [comment]
Closing since it's stale. Feel free to reopen with requested changes.

**Comment by jaspermayone** [comment]
@Luke-Oldenburg let's leave this open, will follow up with @Hex-4.

File: TLDR.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+# Introduction
+
+
+We're Purple Bubble, a crew of teens on a mission to make messaging epic again! Tired of feeling like the product in those other apps? We get it. That's why we're building something new: an app that puts YOU in control. No ads, no data mining, no creepy algorithms. Just you, your friends, and a whole lot of fun.
+
+We're in the super-early stages (think building a pillow fort!), and we're looking for awesome testers to join our inner circle. You can sign up for our newsletter at [purplebubble.org](purplebubble.org) - we promise it'll be way more exciting than your dentist's newsletter. 4-5 updates a year, no spam.
+
+> **Wanna chat?**
+>
+> We mostly communicate through the Purple Bubble Slack (email slack@purplebubble.org or ask someone that's already in the slack for an invite!), but you can also email us (team@purplebubble.org) or open a Github issue.
```
Comment (jaspermayone) [inline TLDR.md]: 2 emails here is redundant!
```suggestion
> We mostly communicate through the Purple Bubble Slack, email slack@purplebubble.org or ask someone that's already in the slack for an invite!
```

**Review by jaspermayone — APPROVED** [review]

File: TLDR.md (hunk 7 above)
Comment (Hex-4) [reply TLDR.md]: ty! should i make a new pr for this?

Comment (jaspermayone) [reply TLDR.md]: Nope! I edited it myself :)

**Pull request merged**